File: BOJ/단계별 문제/11. 브루트 포스/5. 영화감독 숌/string.cpp
// DrMaemi - 2020KB, 56ms
#include <iostream>
#include <string>
using namespace std;

int main() {
    int N, order = 0;

    cin >> N;

    for (int n=666; ; n++) {
        string s = to_string(n);
        int cnt = 0;

        for (char c: s) {
            if (c == '6') {
                cnt++;

                if (cnt == 3) {
                    order++;
                    break;
                }
            }

            else {
                cnt = 0;
            }
        }

        if (order == N) {
            cout << s;
            break;
        }
    }

    return 0;
}

File: BOJ/단계별 문제/11. 브루트 포스/5. 영화감독 숌/string.py
N = int(input())
n = 666
found = 0

while True:
    s = str(n)
    cnt = 0

    for c in s:
        if c == '6':
            cnt += 1

            if cnt == 3:
                found += 1

        else:
            cnt = 0

    if found == N:
        print(s)
        break

    n += 1